api/src/controllers/website.js

```diff
@@ -151,9 +151,28 @@ async function updatePublicationOptions(req, res, next) { // eslint-disable-line
   }
 }
 
+async function updateTitle(req, res, next) { // eslint-disable-line no-unused-vars
+  const updatedTitle = req.body.websiteTitle;
+  const { team_id: _id } = req.params;
+  try {
+    await Website.updateOne(
+      { teamId: _id },
+      {
+        $set: {
+          title: updatedTitle,
+        },
+      },
+    );
```

Comment:
What happens if a team doesn't originally have a title (and so `title` is not an attribute stored under website schema on mongoDB), and they add it later. Does `$set` still work?

Reply:
yup, it will still work.